File: FestivalJava/Networking/src/main/java/dto/TransactionDTO.java
package dto;

import Domain.Show;
import Domain.Transaction;

/**
 * Created by Sebi on 29-Mar-17.
 */
public class TransactionDTO {
    private Integer id;
    private String clientName;
    private Integer numberOfTickets;
    private ShowDTO show;

    public TransactionDTO(Integer id, String clientName,
                       Integer numberOfTickets, ShowDTO show){
        this.id = id;
        this.clientName = clientName;
        this.numberOfTickets = numberOfTickets;
        this.show = show;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getClientName() {
        return clientName;
    }

    public void setClientName(String clientName) {
        this.clientName = clientName;
    }

    public Integer getNumberOfTickets() {
        return numberOfTickets;
    }

    public void setNumberOfTickets(Integer numberOfTickets) {
        this.numberOfTickets = numberOfTickets;
    }

    public ShowDTO getShow() {
        return show;
    }

    public void setShow(ShowDTO show) {
        this.show = show;
    }
}

File: FestivalC#/Festival/Networking/dto/TransactionDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Networking.dto
{
    [Serializable]
    public class TransactionDTO
    {
        private int id;
        private string clientName;
        private int numberOfTickets;
        private ShowDTO show;

        public int Id
        {
            get
            {
                return id;
            }

            set
            {
                id = value;
            }
        }

        public string ClientName
        {
            get
            {
                return clientName;
            }

            set
            {
                clientName = value;
            }
        }

        public int NumberOfTickets
        {
            get
            {
                return numberOfTickets;
            }

            set
            {
                numberOfTickets = value;
            }
        }

        public ShowDTO Show
        {
            get
            {
                return show;
            }

            set
            {
                show = value;
            }
        }

        public TransactionDTO(int id, string clientName,
                           int numberOfTickets, ShowDTO show)
        {
            this.Id = id;
            this.ClientName = clientName;
            this.NumberOfTickets = numberOfTickets;
            this.Show = show;
        }
    }
}
